config/routes.rb

```diff
@@ -27,6 +27,9 @@
       resource :lock, only: %i(create update destroy)
       resources :tasks, only: %i(index show) do
         resource :output, only: :show
+        member do
+          post :abort
+        end
```

Comment:
Not sure if `POST tasks/:id/abort` is the best. REST wise `POST` is supposed to create something.

We could do `PUT tasks/:id/abort`.